### PR TITLE
Jtm reconfig add arm

### PR DESCRIPTION
**Pull request by BigChiefSmidgeums**

### CODE REVIEW

File: app/views/dashboard/arms/_add_arm_form.html.haml [hunk 1]
```diff
@@ -40,10 +40,10 @@
               .col-sm-7= a.text_field "name", {class: 'form-control'}
             .form-group
               = a.label "subject_count", t(:dashboard)[:arms][:subject_count], class: "col-sm-4 control-label"
-              .col-sm-7= a.text_field "subject_count", {class: 'form-control'}
+              .col-sm-7= a.text_field "subject_count", {class: 'form-control', value: ''}
```
Comment (kyle-glick) [inline app/views/dashboard/arms/_add_arm_form.html.haml]: Why add the empty value to this text_field? Shouldn't it already be done?

File: db/schema.rb [hunk 2]
```diff
@@ -1011,7 +1010,7 @@
     t.string   "status",                 limit: 255, default: "Pending"
     t.integer  "approved_by",            limit: 4
     t.datetime "approved_at"
-    t.float    "percent_subsidy",        limit: 24,  default: 0.0
+    t.float    "percent_subsidy",        limit: 24
```
Comment (kyle-glick) [inline db/schema.rb]: Can we confirm this is correct?

Comment (jayhardee9) [inline db/schema.rb]: This disagrees with master.

**Comment by jwiel86** [comment]
looks good!

**Pull request merged**